src/shapes/IText/ITextBehavior.ts

```diff
@@ -148,21 +148,26 @@ export abstract class ITextBehavior<
     });
   }
 
+  /**
+   * changes the cursor from visible to invisible
+   */
   private _tick(delay?: number) {
     this._currentTickState = this._animateCursor({
-      toValue: 1,
-      duration: this.cursorDuration,
-      delay,
+      toValue: 0,
+      duration: this.cursorDuration / 2,
+      delay: delay || 100,
```

Comment:
This means that of I pass 0 to the delay it will be 100.
So use ?? instead of better off make it a default value in the method signature.

Reply:
In that case I would change the param from delay to something else and handle the delay value internally.

Reply:
i think it should me delay: Math.min(delay, 100) so there is no confusion anymore.